src/bin/commands/containers.rs

```diff
@@ -319,9 +318,9 @@ pub fn list_volumes(
     }: ListVolumes,
     engine: &docker::Engine,
 ) -> cross::Result<()> {
-    let msg_info = MessageInfo::create(verbose, quiet, color.as_deref())?;
-    for line in get_cross_volumes(engine, msg_info)?.iter() {
-        shell::print(line, msg_info)?;
+    let mut msg_info = MessageInfo::create(verbose, quiet, color.as_deref())?;
```

Comment:
This needs to be moved outwards: `msg_info` should be able to be passed to `get_engine`.